src/modules/IscsiLioData.rb

```diff
@@ -1319,6 +1320,14 @@ def SetAuth(tgt, tpg, clnt, incoming, outgoing)
           ret = LogExecCmd("#{cmd} \"\" \"\" ") && ret
         end
       end
+      no_authentication=Convert.to_boolean(UI.QueryWidget(Id(:auth_none), :Value))
```

Comment:
You can remove the `Convert.to_boolean` part, this is just a legacy code from the YCP->Ruby conversion. Should not be needed in the new written code.